python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -3347,6 +3347,22 @@ def test_can_create_multiple_sessions_to_different_remotes(self):
             self.assertIn("Create a new SparkSession is only supported with SparkConnect.", str(e))
 
 
+class SparkConnectSessionWithOptionsTest(ReusedConnectTestCase):
+    def setUp(self) -> None:
+        self.spark = (
+            PySparkSession.builder.config("spark.sql.adaptive.enabled", False)
```

Comment:
I think you can just set any config like `config("a", "b")` and `spark.conf.get("a") == "b"`